docs/api.md

```diff
@@ -605,6 +605,7 @@ Shortcut for [`keyboard.down`](#keyboarddownkey-options) and [`keyboard.up`](#ke
         - `y` <[number]> y-coordinate of top-left corner of clip area
         - `width` <[number]> width of clipping area
         - `height` <[number]> height of clipping area
+    - `noBackground` <[boolean]> Hides default white background and allows capturing screenshots with transparency. Defaults to `false`.
```

Comment:
How about calling this `background` which defaults to `true`, or maybe `transparentBackground` which defaults to false? The double negation in `noBackground: false` feels weird.